inferno/run-inferno.sh

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+# Get Config
+CONFIG=`cat config.json`
+BULK_URL=$(echo $CONFIG | jq -r ".bulk_url")
+TOKEN_URL=$(echo $CONFIG | jq -r ".auth.token_endpoint")
+CLIENT_ID=$(echo $CONFIG | jq -r ".auth.client_id")
+CLIENT_SECRET=$(echo $CONFIG | jq -r ".auth.client_secret")
+TESTS=$(echo $CONFIG | jq -r ".required_tests.[]")
```

Comment:
This line only worked on my machine without a period
```suggestion
TESTS=$(echo $CONFIG | jq -r ".required_tests[]")
```

Reply:
this is how I had it, then it wouldn't work unless i put the dot. then it was fine, but i probably changed something else too. but anyways, I removed the dot because ya I think this is correct. good find!